app/controllers/users_controller.rb

```diff
@@ -0,0 +1,23 @@
+class UsersController < ApplicationController
+  def new
+    redirect_to home_path unless !logged_in?

```

Comment:
This is actually not good, - you are redirecting anyone who is not logged in from accessing user registration. If you think about it - who wants to register? And the answer is - people who are _not_ registered at your page and that means they will not be even logged.